homeassistant/components/airzone/entity.py

```diff
@@ -125,7 +126,7 @@ def __init__(
             identifiers={(DOMAIN, f"{entry.entry_id}_ws")},
             manufacturer=MANUFACTURER,
             model=self.get_airzone_value(AZD_MODEL),
-            name=self.get_airzone_value(AZD_FULL_NAME),
```

Comment:
self.get_airzone_value(AZD_FULL_NAME) returns "WebServer 11:22:33:44:55:66" (with MAC address), which helps identify the Airzone WebServer for users with multiple Airzone webservers (example: multiple Aidoo Pro devices).

Reply:
The current tests didn't and I couldn't figure out where to fix this